docs/sources/operators-guide/monitoring-grafana-mimir/collecting-metrics-and-logs.md

```diff
@@ -0,0 +1,335 @@
+---
+title: "Collecting metrics and logs from Grafana Mimir"
+menuTitle: "Collecting metrics and logs"
+description: "Learn how to collect metrics and logs from Grafana Mimir itself"
+weight: 60
+---
+
+# Collecting metrics and logs from Grafana Mimir
+
+This article helps you to collect logs and metrics from a Mimir or GEM cluster. To set up dashboards and alerts,
+see [Installing Grafana Mimir dashboards and alerts]({{< relref "installing-dashboards-and-alerts.md" >}})
+or [Grafana Cloud: Self-hosted Grafana Mimir integration](https://grafana.com/docs/grafana-cloud/integrations/integrations/integration-mimir/)
+.
+
+It is easier and recommended to monitor a cluster if it has been installed via
+the [Grafana Mimir Helm chart](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed). It
+is also possible to use this integration if Mimir has been deployed another way. For more information,
+see [Collect metrics and logs without the Helm chart](#collect-metrics-and-logs-without-the-helm-chart).
+
+## Collect metrics and logs from the Helm chart
+
+To set up the collection of metrics and logs, follow the steps that are based on the version of the Helm chart that has
+been deployed:
+
+* For a stable release:
+    * \>= 3.x.x:
+      See [Collect metrics and logs via Helm chart built-in metamonitoring](#collect-metrics-and-logs-via-the-helm-chart)
+    * \< 3.x.x: See [Collect metrics and logs via Grafana Agent](#collect-metrics-and-logs-via-grafana-agent)
+* For non-Helm installations or installations of the deprecated enterprise-metrics Helm chart, see [Collect metrics and logs without the helm chart](#collect-metrics-and-logs-without-the-helm-chart).
+
+### Collect metrics and logs via the Helm chart
+
+Starting from version `3.0.0`, the Helm chart sends metrics to a Prometheus-compatible server and logs to a Loki
+cluster. The chart can also scrape additional metrics from kube-state-metrics, kubelet, and cAdvisor. node_exporter
```

Comment:
```suggestion
cluster. The chart can also scrape additional metrics from kube-state-metrics, kubelet, and cAdvisor. Any node_exporter
```
Adding a word in front of node_exporter so the sentence does not start with a lowercase letter.

Reply:
Will change to "No node_exporter metrics are collected by the chart."

Reply:
Alternatively, The Helm chart does not collect node_exporter metrics.